### PR TITLE
Section1

### PR DESCRIPTION
**Pull request by wakandajp**
done section 1

but with some bugs, help me~~ (it could be issues after merge this pull request)
- header(logo & site title) should not be fixed but it's fixed on the top
- nav doesn't work when scrolling (each dots are supposed to turn white when scroll down to them)
    - scroll event doesn't work on my environment

### CODE REVIEW

File: public/section1.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        console.log('done3');
+        if (nav3Dot.className === 'inner-circle') {
+            nav3Dot.classList.add("active");
+            nav1Dot.classList.remove("active");
+            nav2Dot.classList.remove("active");
+            window.location.hash='three';
+        }
+    });
+
+    var scrollMenu = function() {
```
Comment (eastend-street) [inline public/section1.js]: Use const instead of var

Comment (wakandajp) [reply public/section1.js]: Thank you
fixed on this commit https://github.com/eastend-street/quote-maker/pull/7/commits/62f4a3752f4702b18915a2b3121f344c7b028f77
not this, this one
https://github.com/eastend-street/quote-maker/pull/7/commits/0958fc578bb0132e3cf8f18d48f03b7ae1ba93c2

Comment (eastend-street) [reply public/section1.js]: Okay, thanks! Wait a moment, still checking other code..

File: public/stylesheets/style.css [hunk 2]
```diff
@@ -185,4 +198,190 @@ img#start-image {
...
+    color: #fafaff;
+    display: flex;
+    align-items: center;
+}
+
+header .wrapper .logo {
+    width: 60px;
+}
+
+header .wrapper .herder-title {
```
Comment (eastend-street) [inline public/stylesheets/style.css]: Typo, herder-title→header-title

Comment (wakandajp) [reply public/stylesheets/style.css]: Thank you!
Fixed in this commit https://github.com/eastend-street/quote-maker/pull/7/commits/59ffc6a397fcb92886deea5f345ea2054c9d9fd2

**Pull request merged**